www/docs/configuration.md

```diff
@@ -97,4 +97,55 @@ If you want to customize, extend or replace the models, you can do this by using
 
 NextAuth.js automatically creates simple, unbranded authentication pages for handling Sign in, Email Verification, callbacks, etc. The options displayed are generated based on the configuration supplied.
 
-@TODO Add example of how to specify custom URLs for pages
\ No newline at end of file
+To add a custom login page, for example. You must add a `pages` array to the config, for example:
+
+```javascript title="/pages/api/auth/[...slug].js"
+  ...
+  pages: {
+    signin: '/auth/signin',
+    signout: '/auth/signout',
+    checkEmail: '/auth/check-email',
+    error: '/auth/error'
+  }
+  ...
+```
+
+Then you must create pages at the above routes. 
+
+In the signin page, for example, in order to get the available providers you must make a request to `/api/auth/providers`.
+
+```jsx title="/pages/auth/signin"
+import React from 'react'
+
+const SignIn = ({ providers }) => {
+  return (
+    <div>
+      {providers && Object.values(providers).map(provider => (
+        <p key={provider.name}>
+          <a href={provider.signinUrl}>
+            <Button id='signin-btn' type='submit' appearance='primary' block>Sign in with {provider.name}</Button>
```

Comment:
```suggestion
            <Button type='submit' appearance='primary' block>Sign in with {provider.name}</Button>
```

Should we drop the ID (or make it `className=`) so the buttons don't all have the same ID?

Reply:
Oh yeah my bad, missed deleting that from copying it out of my project :sweat_smile: